src/components/homepage/chat-overview.tsx

```diff
@@ -1,81 +1,112 @@
 "use client";
 
+import { Input } from "~/components/ui/input";
 import { useQuery } from "convex/react";
 import { api } from "../../../convex/_generated/api";
 import { useUser } from "@clerk/nextjs";
+import { useState } from "react";
 import { Avatar, AvatarFallback } from "~/components/ui/avatar";
+import Badge from "~/components/ui/badge";
 import { NotebookText } from "lucide-react";
-import { Check } from "lucide-react";
-import { MousePointerClick } from "lucide-react";
 
-export function Chats() {
+interface User {
```

Comment:
Instead of this, you can just infer the types from the backend: https://docs.convex.dev/typescript#writing-frontend-code-in-typescript. If you have further questions, you can ask the convex discord channel etc. and if I have time also me.